packages/compass-crud/src/utils/cancellable-queries.spec.js

```diff
@@ -73,6 +74,12 @@ describe('cancellable-queries', function() {
     await insertOne('config.collections', { _id: 'cancel.sharded', key: { a: 1 } }, {});
   });
 
+  after(async function() {
+    if (dataService) {
+      await dataService.disconnect();
```

Comment:
With the `mocha` version update we're now not immediately quitting after tests, so this cleanup ensures we can exit gracefully.